MeshLib/findElementsWithinRadius.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "findElementsWithinRadius.h"
+
+
+#include <deque>
+#include <unordered_set>
+
+#include "BaseLib/uniqueInsert.h"
+#include "MathLib/MathTools.h"
+
+#include "Elements/Element.h"
+#include "Node.h"
+
+namespace MeshLib {
+std::vector<std::size_t> findElementsWithinRadius(Element const& start_element,
+                                                  double const radius)
```

Comment:
Maybe you can pass `sqr_radius` instead of `radius` to omit repeated computation of `radius * radius`?